src/drivers/net/bnxt/bnxt_dbg.h

```diff
@@ -475,7 +475,7 @@ void dbg_rx_stat(struct bnxt *bp)
 #endif
 
 #if defined(DEBUG_CQ)
-static void dump_cq(struct cmpl_base *cmp, u16 cid)
+static void dump_cq(struct cmpl_base *cmp, u16 cid, u8 toggle)
```

Comment:
Neither BCM957608 specific.

Reply:
This new field is specific to BCM57608 functionality

Reply:
Yeah, my original _"Neither BCM957608 specific."_ should have been something like _"That doesn't look BCM957608 specific, why is it here in this commit?"_.

Reply:
Understood.  An additional field was added for completion processing. The existing completion processing was updated to handle this new field, rather than creating a separate function.  Would it be helpful if comments were added into the code to explain this?

Reply:
The question _"why is it here in this commit?"_ reworded as a request (with **why**) _"Please put the debug extension into a separate commit. Have 1 commit to 1 functional change. Make it possible that `git bisect` can identify the exact breaking change. Express that you understand that you are changing existing code that is in actual use."_

Reply:
I apologize if the meaning of my last sentence did not convey my proper meaning.  I really would like your input if comments would help to clarify why this field was added